app/js/index.js

```diff
@@ -0,0 +1,307 @@
+/*jslint plusplus: true */
```

Comment:
I would also put function declaration like `getLocalStorage, createListRow, clearList, clearList, renderList, etc.` to separete file (and module). It would be good, if we have left `index.js` clear and transparent only with business logic.